### ISSUE TITLE
Refresh feed list (OPML)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#8]
```
User should be able to update/refresh the feed list from the same opml
file. Currently it will add the same feed to the list even if it already
exists. Application should check if the feed already exists in the list and
it shouldn't add it if it does.
```

Original issue reported on code.google.com by `tlaukka...@gmail.com` on 25 Nov 2006 at 8:14

**Comment by GoogleCodeExporter** [comment]
```
This feature is implemented in the release 1.3
```

Original comment by `tlaukka...@gmail.com` on 2 Dec 2006 at 9:00
- Added labels: **Milestone-Release1.3**

**Comment by GoogleCodeExporter** [comment]
```
This was fixed in the v1.3 release.
```

Original comment by `tlaukka...@gmail.com` on 17 Dec 2006 at 7:46
- Changed state: **Fixed**
